docs/content/configuration.md

```diff
@@ -733,24 +733,38 @@ func init() {
 
 ### Using private image from OCI repositories
 
-When using a private image from an OCI registry the credentials are mandatory as the OCI downloader needs the credentials for the pull operation.
+When using a private image from an OCI registry you need to specify an authentication method. Supported authentication methods are listed in the [Services](#services) section. The Azure managed identity plugin
+is not supported at this point in time.
 
-Examples of setting credetials for pulling private images:
-*AWS ECR* private image usually requires at least basic authentication. The credentials to authenticate can be obtained using the AWS CLI command `aws ecr get-login` and those can be passed to the service configuration as basic bearer credentials as follows:
+Examples of setting credentials for pulling private images:
+*AWS ECR* private images usually require at least basic authentication. The credentials to authenticate can be obtained using the AWS CLI command `aws ecr get-login` and those can be passed to the service configuration as basic bearer credentials as follows:
+```yaml
+credentials:
+  bearer:
+    scheme: "Basic"
+    token: "<username>:<password>"
 ```
- credentials:
-      bearer:
-        scheme: "Basic"
-        token: "<username>:<password>"
+
+Other ECR authentication methods also work:
```

Comment:
Do you mean other AWS authentication methods?

Reply:
Thanks, good catch!
Addressed in https://github.com/open-policy-agent/opa/pull/5560/commits/99b1490097faf1e9d2d0cf817b1ed00e1cc9a9bd